Do not invent our own users for dante-server, use "proxy" and "nobody".

Index: b/example/sockd.conf
===================================================================
--- a/example/sockd.conf
+++ b/example/sockd.conf
@@ -78,13 +78,16 @@
 # when doing something that can require privilege, it will use the
 # userid "sockd".
 #user.privileged: sockd
+user.privileged: proxy
 
 # when running as usual, it will use the unprivileged userid of "sockd".
 #user.unprivileged: sockd
+user.unprivileged: nobody
 
 # If you compiled with libwrap support, what userid should it use
 # when executing your libwrap commands?  "libwrap".
 #user.libwrap: libwrap
+user.libwrap: nobody
 
 
 #
